### PR TITLE
Implement controlleres

### PR DESCRIPTION
**Pull request by adel-gu**
### For each URL I created:

- A route.
- An action in the correct controller.
- A view file.

### Added code to handle only the 4 required URLs.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root "users#index"
+
+  resources :users do
+    resources :posts
```
Comment (mikethreels) [inline config/routes.rb]: - [x] please make sure to avoid unnecessarily exposing endpoints.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Adel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (adel-gu) [reply config/routes.rb]: Hi, @mikethreels 👋 thanks for the review 👏

Am trying to understand your comments and the only thing that comes to my mind is hiding or removing the line `root "users#index"` is that correct ?? if not kindly provide some information or a link that will help me understand your point.

Once again thank you very much ❤

**Review by uwadonat — APPROVED** [review]
## Hi @adel-gu,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Good pull request message ✔️
- Professional readme file ✔️
- You are hiding unneeded endpoints ✔️

appr

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**